### PR TITLE
Add learning goals for rover tutorials

### PR DESCRIPTION
**Pull request by npentrel**
# Description

Please include a summary of the change and what is does. Please also include relevant motivation and context.

### CODE REVIEW

File: docs/tutorials/services/try-viam-color-detection.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ no_list: true
 # SMEs: Hazal
 ---
 
+<!-- LEARNING GOALS
+After following this tutorial, you will understand how the ML model service and the Vision service work together and you will be able to use both alonside the base and camera components to make a machine respond to the world around it.  -->
```
Comment (sguequierre) [inline docs/tutorials/services/try-viam-color-detection.md:24]:
```suggestion
After following this tutorial, you will understand how the ML model service and the Vision service work together and you will be able to use both alongside the base and camera components to make a machine respond to the world around it.  -->
```

File: docs/tutorials/services/color-detection-scuttle.md [hunk 2]
```diff
@@ -24,6 +24,9 @@ cost: 540
 no_list: true
 ---
 
+<!-- LEARNING GOALS
+After following this tutorial, you will understand how the ML model service and the Vision service work together and you will be able to use both alonside the base and camera components to make a machine respond to the world around it.  -->
```
Comment (sguequierre) [inline docs/tutorials/services/color-detection-scuttle.md:28]:
```suggestion
After following this tutorial, you will understand how the ML model service and the Vision service work together and you will be able to use both alongside the base and camera components to make a machine respond to the world around it.  -->
```

File: docs/tutorials/control/gamepad.md [hunk 3]
```diff
@@ -24,9 +24,16 @@ updated: "2024-04-17"
...
 ---
 
-This tutorial teaches you how to add a Bluetooth gamepad controller to your wheeled robot.
 By the end of this tutorial, you'll be able to drive your rover around like an RC car.
 
+{{< alert title="Learning Goals" color="info" >}}
+
+After following this tutorial, you will be able to:
+
+- use the input controller component to control your machine using a gamepad
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:33]: why is this one bulleted and first not?

Comment (npentrel) [inline docs/tutorials/control/gamepad.md:33]:
```suggestion
After following this tutorial, you will be able to use the input controller component to control your machine using a gamepad
```

**Comment by npentrel** [comment]
updated @sguequierre

File: docs/tutorials/control/gamepad.md [hunk 4]
```diff
@@ -24,9 +24,14 @@ updated: "2024-04-17"
 cost: 575
 ---
 
-This tutorial teaches you how to add a Bluetooth gamepad controller to your wheeled robot.
 By the end of this tutorial, you'll be able to drive your rover around like an RC car.
 
+{{< alert title="Learning Goals" color="info" >}}
+
+After following this tutorial, you will be able to use the input controller component to control your machine using a gamepad
```
Comment (sguequierre) [inline docs/tutorials/control/gamepad.md:31]:
```suggestion
After following this tutorial, you will be able to use the input controller component to control your machine using a gamepad.
```

**Review by sguequierre — APPROVED** [review]
LGTM % 1 suggestion

**Pull request merged**